tests/framework/db/ActiveRecordTest.php

```diff
@@ -940,7 +940,7 @@ public function testJoinWithAlias($aliasMethod)
         /** @var $query ActiveQuery */
         $query = Order::find()->joinWith(['customer c']);
         if ($aliasMethod === 'explicit') {
-            $count = $query->count('c.id');
+            $count = $query->count('[[c.id]]');
```

Comment:
Shouldn't it work w/o specifiyng quoting explicitly?

Reply:
If I remove the explicit quotes in the identifier I will get:

```php
1) yiiunit\framework\db\oci\ActiveRecordTest::testJoinWithAlias with data set #0 ('explicit')
yii\db\Exception: SQLSTATE[HY000]: General error: 904 OCIStmtExecute: ORA-00904: "C"."ID": invalid identifier
 (ext\pdo_oci\oci_statement.c:157)
The SQL being executed was: SELECT COUNT(c.id) FROM "order" LEFT JOIN "customer" "c" ON "order"."customer_id" = "c"."id"
```